### PR TITLE
[SALAD-23113] WebApp - Add demand set up alerts section

### PR DESCRIPTION
**Pull request by Maks19**
**Description:** Add demand set up alerts section

### CODE REVIEW

File: packages/web-app/src/modules/demand-alerts-views/utils.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import type { CSSObjectWithLabel } from "react-select";
```
Comment (vitto-moz) [inline packages/web-app/src/modules/demand-alerts-views/utils.ts:1]: what do you think about to name this file `constants.ts`, looks more relevant considering content

File: packages/web-app/src/modules/demand-alerts-views/utils.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  { label: 'NVIDIA RTX 4070 Ti Super', value: 'NVIDIA RTX 4070 Ti Super' },
+]
+
+export const demandScenarios = [
+  { label: 'High Demand', value: 'highDemand' },
+  { label: 'Moderate Demand', value: 'moderateDemand' },
+  { label: 'Low Demand', value: 'lowDemand' },
+]
+
+export const customSetUpDropdownStyles = {
```
Comment (vitto-moz) [inline packages/web-app/src/modules/demand-alerts-views/utils.ts:16]: why do we need custom styling? 
Could you share the file page/UI where we are going to use it? j

Comment (Maks19) [reply packages/web-app/src/modules/demand-alerts-views/utils.ts:16]: It's used in DemandAlertsSetUp for Dropdowns. This designed was approved

File: packages/web-app/src/components/Dropdown.tsx [hunk 3]
```diff
@@ -1,43 +1,62 @@
...
     fontFamily: theme.fontGroteskBook19,
     fontSize: 12,
-    color: theme.lightGreen,
   },
 })
 interface Option {
   label: string
   value: string
 }
 interface Props extends WithStyles<typeof styles> {
```
Comment (vitto-moz) [inline packages/web-app/src/components/Dropdown.tsx:18]: let's add `DropdownStylesConfig `
```
export type DropdownStylesConfig = StylesConfig<Option | '', false, GroupBase<Option | ''>>

interface Props extends WithStyles<typeof styles> {
  customStyles?: DropdownStylesConfig
  options?: Option[]
  value?: string
  onChange?: (value?: any) => void
}

File: packages/web-app/src/components/Dropdown.tsx [hunk 4]
```diff
@@ -1,43 +1,62 @@
...
   onChange?: (value?: any) => void
 }
 
-const _Dropdown: FC<Props> = ({ classes, options, value, onChange }) => {
+const _Dropdown: FC<Props> = ({ classes, customStyles, options, value, onChange }) => {
   const selectedValue = value && options?.find((option) => option.value === value)
 
-  const customStyles = {
-    option: (styles: any) => {
+  const defaultStyles = {
```
Comment (vitto-moz) [inline packages/web-app/src/components/Dropdown.tsx:28]:
```suggestion
  const defaultStyles: DropdownStylesConfig = {
```

File: packages/web-app/src/modules/demand-alerts-views/utils.ts (hunk 2 above)
Comment (vitto-moz) [inline packages/web-app/src/modules/demand-alerts-views/utils.ts]:
```suggestion
export const customSetUpDropdownStyles: DropdownStylesConfig  = {
```

**Review by vitto-moz — APPROVED** [review]

**Pull request merged**